build.gradle

```diff
@@ -18,13 +18,11 @@ buildscript {
     repositories {
         maven { url "https://plugins.gradle.org/m2/" }
     }
-    dependencies {
-        classpath 'gradle.plugin.org.jetbrains:gradle-intellij-plugin:0.0.43'
-    }
 }
 
 plugins {
     id 'net.researchgate.release' version '2.3.4'
+    id 'org.jetbrains.intellij' version '0.1.5'
```

Comment:
Its the new way to do it since gradle 2.1. I only recently upgraded the gradle wrapper on the project.

Its the version of the plugin that's breaking stuff. Not sure why yet.